docker-compose.yml

```diff
@@ -6,31 +6,28 @@ services:
       context: .
       dockerfile: Dockerfile.template
       args:
-        OS_VERSION: ${OS_VERSION:-2.58.6_rev1}
-        DEVICE_TYPE: ${DEVICE_TYPE:-genericx86-64-ext}
+        OS_VERSION: ${OS_VERSION:-2.48.0_rev3}
+        DEVICE_TYPE: ${DEVICE_TYPE:-intel-nuc}
     privileged: true
     stop_signal: SIGRTMIN+3
     stop_grace_period: 30s
     volumes:
       - /lib/modules:/lib/modules:ro
       - ${CONFIG_JSON:-./config.json}:/var/local/config.json
-      - boot:/mnt/boot
-      - data:/mnt/data
-      - state:/mnt/state
+      - /mnt/boot
+      - /mnt/data
+      - /mnt/state
+      - /sys/fs/cgroup/systemd
     tmpfs:
       - /run
       - /run/lock
       - /tmp
       - /var/lib/journal
-      - /sys/fs/cgroup/systemd
     # this is required for the dns option to have any effect
     # see https://github.com/docker/compose/issues/2847#issuecomment-658999887
     network_mode: bridge
+    extra_hosts:
+        - "api.${BOB_TLD} vpn.${BOB_TLD} registry2.${BOB_TLD} delta.${BOB_TLD} s3.${BOB_TLD}:${BOB_IP}" #host and ip
```

Comment:
same here, we could have a `docker-compose.bob.yml` that contains these modifications

Reply:
I can follow this, thanks. Creating a bob folder and moving docker-compose.yml into bob folder